lib/ExternalModule.js

```diff
@@ -18,6 +18,10 @@ class ExternalModule extends Module {
 		this.external = true;
 	}
 
+	libIdent() {
+		return this.request;
```

Comment:
also use userRequest here.

this is not passed to the module yet:
add a userRequest constructor argumunt and pass it from the ExternalsPlugin